spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  # User model associations Tests.
+  context 'He can build events' do
+    user8 = User.create!(name: 'user8',
+                         email: 'user8@gmail.com',
+                         password: 'password')
+
+    it 'has none to begin with' do
```

Comment:
Our test are failing and let's do some unit test for controller.
![Screenshot 2019-09-06 at 3 03 07 PM](https://user-images.githubusercontent.com/10922508/64433997-7e6a5080-d0b7-11e9-9a20-1185a611e610.png)